### PR TITLE
fix for - Not able to remove or add a new cluster-admin in rosa cli

### PR DESCRIPTION
**Pull request by zvikorn**
fix for - Can't create temporary admin user for ROSA cluster

Signed-off-by: Tzvika Avni <tavni@redhat.com>

[SDA-6170](https://issues.redhat.com//browse/SDA-6170)
[SDA-6106](https://issues.redhat.com//browse/SDA-6106)

### CODE REVIEW

**Comment by zvikorn** [comment]
@pvasant PTAL

**Comment by pvasant** [comment]
@zgalor @oriAdler  PTAL

File: cmd/dlt/admin/cmd.go [hunk 1]
```diff
@@ -75,29 +75,39 @@ func run(cmd *cobra.Command, _ []string) {
 	if confirm.Confirm("delete %s user on cluster %s", idp.ClusterAdminUsername, clusterKey) {
 		// delete `cluster-admin` user from the HTPasswd IDP
 		r.Reporter.Debugf("Deleting user '%s' identity provider on cluster '%s'", idp.ClusterAdminUsername, clusterKey)
-		err = r.OCMClient.DeleteHTPasswdUser(idp.ClusterAdminUsername, cluster.ID(), htpasswdIDP)
+		err = r.OCMClient.DeleteUser(cluster.ID(), "cluster-admins", "cluster-admin")
 		if err != nil {
-			r.Reporter.Errorf("Failed to delete '%s' user from the HTPasswd IDP of cluster '%s': %s",
+			r.Reporter.Errorf("Failed to delete '%s' user from clster-admins groups	 of cluster '%s': %s",
```
Comment (samira-barouti) [inline cmd/dlt/admin/cmd.go:80]: typo:
clstr -> cluster

**Review by samira-barouti — COMMENTED** [review]
A few questions/comments

**Review by samira-barouti — CHANGES_REQUESTED** [review]

**Comment by samira-barouti** [comment]
@zvikorn I left one comment on the name of the constant. Other than that, it is GTG.

File: cmd/grant/user/cmd.go [hunk 2]
```diff
@@ -85,8 +86,8 @@ func run(_ *cobra.Command, argv []string) {
 		)
 		os.Exit(1)
 	}
-	if username == "cluster-admin" {
-		r.Reporter.Errorf("Username 'cluster-admin' is not allowed")
+	if username == idp.ClusterAdminUsername {
```
[the remark this answers is not in the document]
Comment (zvikorn) [reply cmd/grant/user/cmd.go:89]: this one as to be exposed as other packages are using it

**Pull request merged**